_build/data/transport.core.system_settings.php

```diff
@@ -706,7 +706,7 @@
 $settings['hidemenu_default']= $xpdo->newObject(modSystemSetting::class);
 $settings['hidemenu_default']->fromArray([
   'key' => 'hidemenu_default',
-  'value' => false,
+  'value' => true,
```

Comment:
Why do we want to hide resources from the menu's by default?

Reply:
We absolutely do not want that.